variables.tf

```diff
@@ -336,6 +336,11 @@ variable "upgrade_pause_time" {
     default = "90"
 }
 
+variable "upgrade_image" {
+    description = "Upgrade image having required image availble for cluster upgrade (quay.io/openshift-release-dev/ocp-release-nightly@,quay.io/openshift-release-dev/ocp-release@ ) eg. quay.io/openshift-release-dev/ocp-release-nightly@sha256:xxxxx"
```

Comment:
Let's use the same description as in ocp4-playbooks.

Reply:
Updated description same as in ocp4-playbooks -  "OCP upgrade image e.g. quay.io/openshift-release-dev/ocp-release-nightly@sha256:xxxxx"